next.config.js

```diff
@@ -9,8 +9,8 @@ const moduleExports = {
   },
 };
 
-const sentryWebpackPluginOptions = {
-  silent: true, // Suppresses all logs
-};
+// const sentryWebpackPluginOptions = {
```

Comment:
Is this not necessary to keep sentry working?

Reply:
yes, it is